doc/psidk/src/run-infrastructure/cli/psibase-create-snapshot.md

```diff
@@ -0,0 +1,31 @@
+# psibase-create-snapshot
+
+## NAME
+
+psibase-create-snapshot - Write a snapshot to a file
+
+## SYNOPSIS
+
+`psibase` `create-snapshot` [`-b`] [`-s` *key-file*\.\.\.] *database* *snapshot-file*
+
+## DESCRIPTION
+
+`psibase create-snapshot` writes a snapshot of the chain state to a file. The snapshot will hold the the most recent state that is signed by the block producers.
+
+## OPTIONS
+
+- `-b`, `--blocks`
+
+  Include the block log in the snapshot. This will make the block log available for queries.
+
+- `-h`, `--help`
+
+  Prints a help message
+
+- `-s`, `--sign` *key-file*
+
+  Signs the snapshot with this key. The file should contain a PKCS #8 private key.
```

Comment:
PEM encoded?

Reply:
At the moment, yes, though I'd eventually like it to handle BER as well.